tools/start-service.sh

```diff
@@ -6,16 +6,7 @@ set -e
 ROOT="$(cd "$(dirname "${BASH_SOURCE[0]:-$0}")" && cd .. && pwd)"
 cd "$ROOT"
 
-# Check whether to use "python" or "python3"
-python_cmd=$(command -v python3 &> /dev/null && echo "python3" || echo "python")
-
-# Activate virtual environment
-if [ ! -d "${ROOT}/.venv" ]; then
-    echo "${ROOT}/.venv directory not found"
-    exit 1
-else
-    source ${ROOT}/.venv/bin/activate
-fi
+./tools/activate-venv.sh
```

Comment:
the file is empty

Reply:
what file is empty? activate-venv.sh? it is not, as far as i can tell.

Reply:
the code in this script were redundant to what activate-venv.sh has, so i replaced it with a call to activate-venv.sh

Reply:
ah! i see it now. it says it is empty, but it is not. i didn't change the contents, only `chmod +x`ed it. it looks like a github bug.

Reply:
when I click on tools/activate-venv.sh I see an empty file:

<img width="237" alt="image" src="https://github.com/microsoft/nodeengine/assets/371009/29d64c89-2a92-42d4-bb84-02a8e09325f7">

Reply:
oh weird, I checked out the PR and I see the content